script/ci.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set +x
+wget https://github.com/mozilla/geckodriver/releases/download/v0.18.0/geckodriver-v0.18.0-linux64.tar.gz -O /tmp/geckodriver.tar.gz && \
+  tar xvf /tmp/geckodriver.tar.gz -C /opt && \
+  ln -fs /opt/geckodriver /usr/bin/geckodriver
+
+apt-get update && \
+  apt-get install -y xvfb firefox-esr
+
+firefox --version
+geckodriver --version
+
+xvfb-run bundle exec rake
```

Comment:
aha! this makes sense! we had to do same thing originally for Volt https://github.com/artsy/volt/pull/2989#discussion-diff-181741086R26 that was fixed by switching to headless chrome later.